Caboodle/Battery/Battery.shared.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Battery
+    {
+        static event BatteryChangedEventHandler BatteryChanagedInternal;
+
+        public delegate void BatteryChangedEventHandler(BatteryChangedEventArgs e);
```

Comment:
Maybe this delegate shouldn't be nested in a class. It will be long to type:

    new Battery.BatteryChangedEventHandler()